src/clients/HubPoolClient.ts

```diff
@@ -691,7 +720,7 @@ export class HubPoolClient {
     this.firstBlockToSearch = update.searchEndBlock + 1; // Next iteration should start off from where this one ended.
 
     this.isUpdated = true;
-    this.logger.debug({ at: "HubPoolClient", message: "HubPool client updated!" });
+    this.logger.debug({ at: "HubPoolClient::update", message: "HubPool client updated!", endBlock: latestBlockNumber });
```

Comment:
Nit: I think @nicholaspai used `#` instead of `::` in the relayer code for specifying where a log was coming from. Should we keep that format?

Reply:
Yeah, I've seen both in use in the codebase. I think @mrice32 tends for `::`, but we definitely have `#` in places. I'm not strongly opinionated, but I do find it helpful when the method name is included in the `at` field.

Reply:
Yeah, this is just an inconsistency between myself and Nick. I think either way is fine unless we're planning to parse these logs programmatically, since we all know what they mean. Also happy to just go with whichever y'all prefer going forward!